### ISSUE TITLE
Maya crashes when running dwpicker.show()

### ISSUE THREAD

**Issue by 54otaku** [DreamWall-Animation/dwpicker#72]
After I put the dwpicker folder into the maya script folder, my maya will crash when I run dwpicker.show() without any error prompts. I am using the windows version of maya2020.4. How can I locate the problem?

**Comment by luckylyk** [comment]
Hi, sad to read that :/

By crash you mean: instantly shutdown ?
1. have you tried in a virgin environment (without any other external script installed).
2. unload all the plugin loaded.

you can try to reset the maya workspace as well.
And if it still doesn't change anything, you can copy the crash log here. We can hopefully get some precious information there.

**Comment by 54otaku** [comment]
Thank you for your reply. The crash I described does refer to an instant shutdown. Maya pauses for 2-3 seconds after running dwpicker.show() and the process disappears. There is no error message, and I don't know where to copy the crash log. I have also tried the two suggestions you mentioned, and even uninstalled and reinstalled Maya, but the problem still exists.Is there any other way I can locate the cause?
